gradient-private-cloud/setup/bare-metal-vms.md

```diff
@@ -22,41 +24,27 @@ The following is an example of how the added line appears in the JSON file. Do n
 }
 ```
 
-* Ensure your SSH user has access to the docker group in /etc/group:
+For each node, you must also:
+
+* Ensure your SSH user has access to the docker group in `/etc/group`:
 
 ```text
-docker:x:999:ubuntu
+docker:x:999:your-user
 ```
 
 * Ensure your SSH public key is installed on each host
-* Ensure sudo is enabled for the account you're logging into
-* Ensure /etc/sshd/sshd\_config has the following setting \(and reload: service ssh reload\)
+* Ensure `sudo` is enabled for the account you're logging into
+* Ensure `/etc/ssh/sshd\_config` has the following setting \(and then reload it by running `service ssh reload`\)
 
 ```text
 AllowTcpForwarding yes
 ```
 
-#### Configuration
-
-Next you should create a file in the gradient-cluster folder you created – the file must be named `main.tf` and should contain the text in the box below \(note the copy icon in the upper right corner\).
-
-Be sure to replace the following fields with the appropriate values:
+#### Terraform Configuration: main.tf
 
-* name \(the same name used when registering the new cluster in the Paperspace web console\)
-* artifacts\_access\_key\_id \(the key for the bucket that was set up for artifacts storage\)
-* artifacts\_path \(the full s3 path to the bucket\)
-* artifacts\_secret\_access\_key
-* cpu\_selector \(node selector to run CPU workloads, defaults to "metal-cpu"\)
-* cluster\_apikey \(provided during registration of the new cluster\)
-* cluster\_handle \(provided during registration of the new cluster\)
-* domain \(same as what was entered during cluster registration\)
-* gpu\_selector \(node selector to run GPU workloads, defaults to "metal-gpu"\)
-* master\_ip1, worker\_ip1, worker\_ip2 \(see below for IP networking info\)
-* shared\_storage\_path and shared\_storage\_server \(see below for NFS info\)
-* ssh\_key\_path \(for the key whose public key is on the nodes being configured\)
-* ssh\_user \(a ssh user who has the above public key in its authorized\_keys file\)
+Next, create a `main.tf` file within your local `gradient-cluster` directory that you created; `main.tf will be a sibling file to the `backend.tf` file that you may have created already. Note: this file _must_ be named `main.tf` since Terraform looks for this configuration file by name.
```

Comment:
```suggestion
Next, create a `main.tf` file within your local `gradient-cluster` directory that you created; `main.tf` will be a sibling file to the `backend.tf` file that you may have created already. Note: this file _must_ be named `main.tf` since Terraform looks for this configuration file by name.
```